### ISSUE TITLE
Tabs Scrolling: Visible tabs won't update after closing a tab

### ISSUE THREAD

**Issue by neikeq** [godotengine/godot#3128]
Steps to recreate:

1. Open many tabs until the arrows appear.
2. Click on the right arrow until you get to the last tab.
3. Close all visible tabs.
4. When you are done, no tabs will be displayed and you will have to click the left arrow to get back to the first tab/tabs that were hidden after step 2.

The tab container should automatically try to fill the free space with open tabs after closing one.

**Comment by neikeq** [comment]
It would also be good if it scrolled to the newly open tabs when arrows are visible.

**Comment by neikeq** [comment]
This issue is not fixed. However, the behaviour is different now. After closing two tabs, all of remaining tabs are hidden (may be related to issue #3314).
![gif](https://cloud.githubusercontent.com/assets/7718100/12296340/5a6d7b82-ba07-11e5-9645-1e2a8a238635.gif)

**Comment by akien-mga** [comment]
Actually it's more #3314 that should be reopened IMO.

**Comment by lmjk** [comment]
The hidden leftmost tabs become inaccessible again on 3.0 rc2, if the tab region gets overcrowded and visible tabs are (re)moved:
![crowded_tab_region_a_30rc2](https://user-images.githubusercontent.com/25223253/35246850-db48bb8e-ffd0-11e7-804e-f9b358887b97.png)
![crowded_tab_region_b_30rc2](https://user-images.githubusercontent.com/25223253/35246851-dbb01acc-ffd0-11e7-8914-ebeeced8f02c.png)
![crowded_tab_region_c_30rc2](https://user-images.githubusercontent.com/25223253/35246853-dc0c9ff4-ffd0-11e7-9f78-35e91b816719.png)

**Comment by akien-mga** [comment]
Please open a new issue, this one has been fixed 2 years ago.

**Comment by lmjk** [comment]
Okay, I'll do it. I thought these were too similar not to be related, sorry!
